code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -2141,3 +2141,18 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 /mob/living/carbon/human/proc/apply_offstation_roles(source)
 	SIGNAL_HANDLER
 	mind.offstation_role = TRUE
+
+/mob/living/carbon/human/proc/handle_light_adjustment()
```

Comment:
This is never called is it?

Reply:
That's on me for not marking the PR as draft, I'm still figuring out how and where to add a signal whenever we need darksight to kick in